openmdao/core/system.py

```diff
@@ -191,14 +210,20 @@ def _setup_paths(self, parent_path):
         parent_path : str
             The pathname of the parent `System`, which is to be prepended to the
             name of this child `System`.
+
+        probdata : `_ProbData`
+            Problem level data container.
         """
-        self._fd_params = None
+        self._reset()
 
         if parent_path:
             self.pathname = '.'.join((parent_path, self.name))
         else:
             self.pathname = self.name
 
+        self._sysdata = _SysData(self.pathname)
```

Comment:
I think I needed to add that early one in order to not break some of our tests that don't do a proper setup.